polylogue/importers/chatgpt.py

```
@@ -9,8 +9,28 @@
 from typing import Any, Dict, Iterable, List, Optional, Set, Tuple
 
 import ijson
+from decimal import Decimal
 
 from ..db import get_raw_import_by_conversation, open_connection
+
+
+class _DecimalEncoder(json.JSONEncoder):
+    """JSON encoder that converts Decimal to float for ijson compatibility."""
+    def default(self, o):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `default`

(ANN202)